examples/with-next-app/app/layout.tsx

```diff
@@ -0,0 +1,76 @@
+'use client';
```

Comment:
also avoid `"use client";` here and encapsulate client dependants - good example of this is on create-wagmi: https://github.com/wagmi-dev/create-wagmi/tree/main/templates/next/rainbowkit/src/app

Reply:
@jxom Great advice; have adopted the Wagmi `providers.tsx` approach.